exporter/datasetexporter/logs_exporter_test.go

```diff
@@ -229,6 +228,65 @@ func TestBuildEventFromLog(t *testing.T) {
 	assert.Equal(t, expected, was)
 }
 
+func TestBuildEventFromLogEventWithoutTimestampWithObservedTimestampUseObservedTimestamp(t *testing.T) {
```

Comment:
Ideally we structure our tests in GIVEN, WHEN and THEN section to increase readability

Reply:
I'm fine with it as long as it's consistent across the board (and also in line with go conventions / best practices) :)